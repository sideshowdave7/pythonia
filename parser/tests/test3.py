c = 2
a = 3 + c
try:
 c
except:
 d
def f(a, b):
 pass

