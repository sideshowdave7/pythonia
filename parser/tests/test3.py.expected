(program
 (= (c) 2)
 (= (a) (arith 3 ("+" c)))
 (try (suite (expr c)) (((except) (suite (expr d)))) #f #f)
 (def (f a b) (suite (pass))))

